workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerCreation/SendCreatePlayerRequestSystem.cs

```diff
@@ -37,15 +37,8 @@ protected override void OnCreateManager()
         public void RequestPlayerCreation(byte[] serializedArguments = null)
         {
             serializedArgumentsCache = serializedArguments;
-
-            var request = new CreatePlayerRequestType();
-            if (serializedArguments != null)
-            {
-                request.SerializedArguments = serializedArguments;
-            }
-
+            var request = new CreatePlayerRequestType(serializedArguments);
```

Comment:
I don't think we handle null assignments in the code generator. As in we don't translate null to `new byte[] { }`

Reply:
serialization of `CreatePlayerRequestType` calls `AddBytes` which has a special case if the value is null :)

(double checked it works when testing locally)